cupy/testing/random.py

```diff
@@ -0,0 +1,60 @@
+import atexit
+import numpy
+
+import cupy
+
+
+# In some tests (which utilize condition.repeat or condition.retry),
+# setUp/tearDown is nested. setup_random() and teardown_random() do their
+# work only in the outermost setUp/tearDown pair.
+_nest_count = 0
+
+
+@atexit.register
+def _check_teardown():
+    assert _nest_count == 0, ('setup_random() and teardown_random() '
+                              'must be called in pairs.')
+
+
+def setup_random(numpy_seed=None, cupy_seed=None):
```

Comment:
No, they're currently not needed, I don't come up with any situation they do.
Should I remove them?